web/src/components/contact/ContactDialog.vue

```diff
@@ -97,7 +99,7 @@ export default {
     form: {
       name: '',
       email: '',
-      message: '',
+      message: 'Please introduce yourself to the speaker, providing details of your event (who, what, when, where), and how they might be a good match.',
```

Comment:
Same here. Also please keep lines within 100 characters wide.

Reply:
For this one, I have tried to use the $t filter but it gives an error in the data() 
Do you know if I should be creating a computed property and then calling that in the data or approaching the issue in a different way?

Reply:
What's the error you see? There should be no need to add a computed property. You'll need to add the raw string to en.json to match the tag called in the `$t`function. Please check out [the docs](http://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting) for more information on how it works.

Or if it's still not making sense, let's do a quick pair 🍐  programming sometime!

Reply:
For this one, I needed to slightly modify the data function to be able to use i18n. All working now :) 

data: (vm) => ({
    form: {
      name: '',
      email: '',
      message: vm.$t('contact.message-default'),
    },
  }),